api_v2/package.json

```
@@ -0,0 +1,77 @@
+{
+  "name": "api_v2",
+  "version": "0.0.1",
+  "description": "",
+  "author": "",
+  "private": true,
+  "license": "UNLICENSED",
+  "scripts": {
+    "build": "nest build",
+    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "start": "nest start",
+    "start:dev": "nest start --watch",
+    "start:debug": "nest start --debug --watch",
```

its good to have short command when running dev environment
```suggestion
    "dev": "nest start --watch",
    "dev:debug": "nest start --debug --watch",
```